### ISSUE TITLE
most segmentDisplays have no crafting recipe

### ISSUE THREAD

**Issue by moTechPlz** [Victorious3/Integrated-Circuits#47]
Expect for the non FMB white version. bytheway. Maybe you can toss in a Hexadecimal mode, maximally displaying FFFF when 4 are connected to each other.

**Comment by Victorious3** [comment]
Don't know what happened to the crafting recipes, I did test them at least once but that was a while ago... I'll get the hex mode in with the next non-bugix release

**Comment by moTechPlz** [comment]
Crafting does work so they the recipies are in. More specifically, NEI does not list them with 'r' except for the base white one. If i press 'r' for instance on orangeWood i get the orange Dye + wool recipe. Ofcourse the dyeing recipies are a noBrainer but more for the sake of completeness. The FMP versions  with the slaps around are a little less obvious.

**Comment by Victorious3** [comment]
Yes, that's a known issue. These recipes are of a custom type, I'd need a NEI plugin for them. Definitely something for a later version,.

**Comment by skyem123** [comment]
@Victorious3, 0.9 has been out for a long time, yet this issue isn't closed!
